recipes-extended/xen/files/libxl-xenmgr-support.patch

```diff
@@ -396,22 +396,6 @@ Index: xen-4.6.4/tools/libxl/xl_cmdimpl.c
      rc = libxl_domain_destroy(ctx, domid, 0);
      if (rc) { fprintf(stderr,"destroy failed (rc=%d)\n",rc); exit(-1); }
      libxl_update_state_direct(ctx, uuid, "shutdown");
-Index: xen-4.6.4/tools/libxl/libxl_device.c
-===================================================================
---- xen-4.6.4.orig/tools/libxl/libxl_device.c
-+++ xen-4.6.4/tools/libxl/libxl_device.c
-@@ -904,6 +904,11 @@ void libxl__initiate_device_remove(libxl
-             }
-         }
- 
-+        if(!libxl__xs_read_checked(gc, t, libxl__device_backend_path(gc, aodev->dev), &state))
```

Comment:
It should be cleaned up in libxl_device.c:libxl__device_destroy().  I think this specific hunk was a mistake to have included in the first place.  I'm pretty sure it was something I added, but I don't have the history or any comments to know for sure.